vaadin-grid-flow/src/main/java/com/vaadin/flow/component/grid/dataview/GridDataViewImpl.java

```diff
@@ -69,4 +70,10 @@ public T getItemOnRow(int rowIndex) {
         }
         return getItems().skip(rowIndex).findFirst().orElse(null);
     }
+
+    @Override
+    public void setIdentifierProvider(IdentifierProvider<T> identifierProvider) {
```

Comment:
This should also be added to `GridListDataView` ? 

(I will need to add it to `GridLazyDataView` too)

Reply:
Done